components/heroku_app/variables.tf

```diff
@@ -0,0 +1,71 @@
+# Required variables:
+variable "environment" {
+  description = "The environment for this application: development, qa, or production."
+}
+
+variable "pipeline" {
+  description = "The Heroku pipeline ID this application should be created in."
+}
+
+variable "name" {
+  description = "The application name."
+}
+
+variable "framework" {
+  description = "The framework for this application (e.g. 'laravel', 'express')."
+}
+
+variable "papertrail_destination" {
+  description = "The Papertrail log destination for this application."
+}
+
+variable "domain" {
+  description = "The domain this application will be accessible at, e.g. longshot.dosomething.org"
```

Comment:
Not a huge deal, but maybe switch this to use a current example since `Longshot` is no more? Maybe `identity.dosomething.org`?

Reply:
Changed in 961fcd1!